docs/eventing/features/authorization.md

```diff
@@ -0,0 +1,471 @@
+kind: EventPolicy
+metadata:
+  name: my-event-policy
+  namespace: default
+spec:
+  to:
+    - ref:
+        apiVersion: eventing.knative.dev/v1
+        kind: Broker
+        name: my-broker
+    - selector:
+        apiVersion: eventing.knative.dev/v1
+        kind: Broker
+        matchLabels:
+          app: special-app
+  from:
+    - ref:
+        apiVersion: sources.knative.dev/v1
+        kind: PingSource
+        name: my-source
+        namespace: another-namespace
+    - sub: system:serviceaccount:default:trusted-app
+    - sub: "system:serviceaccount:default:other-*"
+  filters:
+    - cesql: "type IN ('order.created', 'order.updated', 'order.canceled')"
+    - exact:
+        type: com.github.push
+```
+
+### Specify for who the `EventPolicy` applies
+
+The `.spec.to` section specifies **where** the events are allowed to be sent. This field is optional; if left empty, the policy applies to all resources within the namespace.
+
+There are two ways to define these targets:
+
+1. `to.ref`:
+    * **Definition**: Directly references a specific resource.
+    * **Example**: In the `EventPolicy` above, the `my-broker` Broker is directly referenced. This means the `EventPolicy` applies to this specific `Broker`.
+    * **Use Case**: Use `to.ref` when you want to protect a specific resource by name.
+    ```yaml
+    to:
+      - ref:
+          apiVersion: eventing.knative.dev/v1
+          kind: Broker
+          name: my-broker
+    ```
+2. `to.selector`:
+    * **Definition**: Uses a label selector to match multiple resources of a specific type.
+    * **Example**: The `EventPolicy` includes a `Broker` with labels matching `app: special-app`. This means the `EventPolicy` applies to all `Brokers` with these labels.
+    * **Use Case**: Use `to.selector` when you want the `EventPolicy` to apply to a group of resources that share common labels.
+
+    ```yaml
+    to:
+      - selector:
+          apiVersion: eventing.knative.dev/v1
+          kind: Broker
+          matchLabels:
+            app: special-app
+    ```
+
+### Specify who is allowed to send events
+
+The `.spec.from` section specifies **who** is allowed to send events to the targets defined in `.spec.to`. There are two ways to define these sources:
+
+1. `from.ref`:
+    * **Definition**: Directly references a specific event source resource.
+    * **Example**: The `my-source` `PingSource` in `another-namespace` is referenced, meaning this specific source is allowed to send events.
+    * **Use Case**: Use `from.ref` when you want to authorize a specific event source.
+    ```yaml
+    from:
+      - ref:
+          apiVersion: sources.knative.dev/v1
+          kind: PingSource
+          name: my-source
+          namespace: another-namespace
+    ```
+
+2. `from.sub`:
+
+    * **Definition**: Specifies a subject, such as a service account, that is allowed to send events. It can include wildcard patterns as a postfix (`*`) for broader matching.
```

Comment:
Is there any other subject other than a service account?

Reply:
Not yet. But when we integrate this with other IDPs and allow sending events from outside the mesh, this would be more relevant.
But to not confuse, I'd better remove it